school_library_decorator/person.rb

```diff
@@ -4,7 +4,7 @@ class Person
   attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] When I run option 3, to create a person and then choose to create a teacher, it throws an error that says wrong number argument given. Please update the parameter of this method so it can better reflect the arguments passed in.

Reply:
Thank you for this valuable review. ❤️ 

The issue has been resolved in the latest comment [e10783f](https://github.com/AshakaE/mini-oop/commit/e10783f134fc51496033e8e21c7f38c824532177). 🟢